docs/releases/v13/version-v13.0.0.md

```diff
@@ -92,7 +92,27 @@ When a record is shared, its inclusion requests will be also accessible. There i
 
 To be completed
 
-### Collections
+### Communities
+
+#### Themed communities
```

Comment:
What is the feature name we want to converge on: "Branded communities" per https://github.com/inveniosoftware/docs-invenio-rdm/pull/795 or "Themed communities"?

Once the other PR is merged, we should add a link to its relevant page(s).

Reply:
not sure, ping: @ntarocco @slint

Reply:
@fenekku thanks for the feedback, does this mean that we exclude "Themed"? I kind of like the simplicity of "Themed communities", on top of what all of you already described above.
"Bespoke" sounds very commercial to me :) Lately it is used by many companies advertising their products.
And Customized/Tailored feel a bit too generic, while "Themed" is immediately conveying the idea that some UI/colors/logo can be changed.

Reply:
I think "Themed" as well is the best in the end. (People found it fine too) With the explanation by Alex and the echo to the theme section I mentioned, it'd go for "Themed Communities".